spec/support/acceptance/helpers/step_helpers.rb

```diff
@@ -58,8 +58,11 @@ def run_n_unit_test_suite
 
     def create_rails_application
       fs.clean
-
-      command = "bundle exec rails new #{fs.project_directory} --skip-bundle --no-rc"
+      if rails_version.to_s.start_with?('6.0')
```

Comment:
`rails_version` is a special object for which `=~` is implemented such that you can use version comparison (like you'd use in a Gemfile) instead of matching on strings. What do you think about saying:

``` ruby
if rails_version =~ '~> 6.0'
```

Reply:
That sounds great thanks. I'll make the change.